source/appendices/e-audit-requirements-and-tests/02-general-tests.rst

```diff
@@ -0,0 +1,85 @@
+
+
+E.2.3 Audit Tests for Double-Click Filtering
+""""""""""""""""""""""""""""""""""""""""""""
+
+This audit test applies to investigations and requests metrics across all COUNTER Reports and should represent the scope of the platform. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor should represent each of those Data_Types proportionately in the audit test.
+
+The test consists of making requests to an item twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second request MUST be recorded, resulting in 1 Total_Item_Investigation and 1 Total_Item_Request. If the two clicks occur with more than 30 seconds between them, then 2 Total_Item_Investigations and 2 Total_Item_Requests must be counted. In both cases only 1 Unique_Item_Investigation and 1 Unique_Item_Request will be reported.
+
+The auditor MUST carry out a total of 30 tests:
+
+* 15 “Inside” tests, whereby 2 identical requests are made and the second request is within 30 seconds of the first.
+* 15 “Outside” tests, whereby 2 identical requests are made and the second request is more than 30 seconds after the first.
+
+The “Inside” tests MUST result in 15 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and the “Outside” tests MUST result in 30 Total_Item_Investigations, 30 Total_Item_Requests, 15 Unique_Item_Investigations and 15 Unique_Item_Requests, for a total of 45 Total_Item_Investigations, 45 Total_Item_Requests, 30 Unique_Item_Investigations and 30 Unique_Item_Requests.
+
+
+E.2.4 Audit Tests for Denials
+"""""""""""""""""""""""""""""
+
+Report providers operating platforms where turnaways or denials are in operation MUST be subject to audit tests for denials. For report providers operating multiple platforms, the audit scope as defined in :numref:`audit` MUST include platforms where turnaways or denials are in operation. Where either Limit_Exceeded or No_License denials do not apply to a report provider, auditors MUST note this in the audit report. This does not require an exemption from the COUNTER Project Director.
+
+These audit tests apply to denial metrics across all COUNTER Reports and should represent the scope of the platform under audit. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor SHOULD represent each of those Data_Types proportionately in the audit test.
+
+
+E.2.4.1 Limit_Exceeded
+''''''''''''''''''''''
+
+Note that the account used for this testing MUST have concurrent / simultaneous-user limit (concurrency limits) set at a single user. A second user attempting to access the database would be denied.
+
+**Option 1**: The report provider denies the user access when the concurrency limit is exceeded upon login.
```

Comment:
Limit_Exceeded is only defined for databases and items, not for platforms, so there is no way to include these denials in a report! Since this has been this way since R4, I'm wondering if there are actually platforms where this option would be applicable?